project-base/app/config/services.yaml

```diff
@@ -253,9 +253,6 @@ services:
     Shopsys\FrameworkBundle\Component\Setting\Setting:
         alias: App\Component\Setting\Setting
 
-    Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileFacade:
-        alias: App\Component\UploadedFile\UploadedFileFacade
```

Comment:
`App\Component\UploadedFile\UploadedFileFacade` still exists, so the alias should remain, or?

Reply:
Good catch, thx!